ansible/manila-client.yml

```diff
@@ -19,9 +20,11 @@
       os_manila_mount_host: "{{ hostvars['localhost']['os_manila_mount_host'] }}"
       os_manila_mount_access_key: "{{ hostvars['localhost']['os_manila_mount_access_key'] }}"
       os_manila_mount_export: "{{ hostvars['localhost']['os_manila_mount_export'] }}"
+      os_manila_mount_share_user: "{{ ceph_mount_share_user }}"
```

Comment:
As above.

Reply:
is os_manila_mount_share_user: "{{ os_manila_mount_share_user }}" valid here?

Reply:
As above :)